src/reducers/resources.js

```diff
@@ -1,5 +1,4 @@
 import _ from "lodash"
-import { resourceEditErrorKey } from "components/editor/Editor"
```

Comment:
This was causing the Editor to be loaded in running tests, which was leading to an open handle. Why did this become a problem when JSON-LD library was updated? No idea whatsoever.